Quaver.Shared/Screens/Results/UI/Header/Contents/ResultsScreenHeaderAvatar.cs

```diff
@@ -50,13 +52,18 @@ public ResultsScreenHeaderAvatar(float size, Bindable<ScoreProcessor> processor)
                     var tex = SteamManager.UserAvatarsLarge[steamId];
 
                     if (tex != UserInterface.UnknownAvatar && SteamManager.UserAvatarsLarge.ContainsKey(steamId))
-                        Avatar.AvatarSprite.Image = tex;
+                        Avatar.Image = tex;
                     else if (tex == UserInterface.UnknownAvatar && SteamManager.UserAvatars.ContainsKey(steamId))
-                        Avatar.AvatarSprite.Image = SteamManager.UserAvatars[steamId];
+                        Avatar.Image = SteamManager.UserAvatars[steamId];
                     else
-                        Avatar.AvatarSprite.Image = UserInterface.UnknownAvatar;
+                        Avatar.Image = UserInterface.UnknownAvatar;
                 }
             }
+
+            GameBase.Game.ScheduledRenderTargetDraws.Add(() =>
+            {
+                Avatar.Image = SpriteAlphaMaskBlend.PerformBlend(Avatar.Image, SkinManager.Skin?.Results?.ResultsAvatarMask ?? UserInterface.ResultsAvatarMask);
```

Comment:
Split into two lines. It's too long.
```cs
var mask = SkinManager.Skin?.Results?.ResultsAvatarMask ?? UserInterface.ResultsAvatarMask;
Avatar.Image = SpriteAlphaMaskBlend.PerformBlend(Avatar.Image, mask);
```
Also, see my comment [here](https://github.com/Quaver/Wobble/pull/120#discussion_r825487367). Memory leak needs to be fixed. Either by caching the avatar or disposing it in `Destroy()`. The latter would probably be a better option if people are trying to modify their skin.